pkg/api/login.go

```diff
@@ -254,6 +254,7 @@ func (hs *HTTPServer) Logout(c *contextmodel.ReqContext) {
 	if err != nil {
 		hs.log.Error("Failed perform proper logout", "error", err)
 		c.Redirect(hs.Cfg.AppSubURL + "/login")
+		return
```

Comment:
Panic logs displayed in issues was from here. When something wen't wrong we did not return here and on line 262 we tried to use `redirect.URL` de-referencing a pointer